tests/performance_tests/NodeosPluginArgs/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/__init__.py ${CMAKE_CURRENT_BINARY_DIR}/__init__.py COPYONLY)
```

Comment:
Default behavior for `configure_file()`  is current source to current binary directory, so for new CMake files, we're omitting the redundant paths.  See `TestHarness` CMakeLists.txt for example.

Reply:
Addressed here: https://github.com/AntelopeIO/leap/pull/546/commits/52b7903d92b667fd836e97aa649c3021ba4c3f72